test/urlRouterSpec.js

```diff
@@ -59,7 +59,31 @@ describe("UrlRouter", function () {
           match = ['/foo/bar', $match];
         }).when('/bar', function($match) {
           match = ['/bar', $match];
-        });
+        }).when('/', function() { 
```

Comment:
These rule definitions may seem abundant - but it was the only way I could consistently reproduce the stable sorting issue in Chrome.